src/docdb/tree/DocDBTreeItemBase.ts

```diff
@@ -33,7 +37,12 @@ export abstract class DocDBTreeItemBase<T> implements IAzureParentTreeItem {
     }
 
     public getDocumentClient(): DocumentClient {
-        return new DocumentClient(this.documentEndpoint, { masterKey: this.masterKey });
+        const documentBase = lib.DocumentBase;
+        var connectionPolicy = new documentBase.ConnectionPolicy();
+        const userPreferenceSSLVerify: boolean = vscode.workspace.getConfiguration().get("http.proxyStrictSSL");
+        connectionPolicy.DisableSSLVerification = !userPreferenceSSLVerify && !this.SSLVerify;
```

Comment:
This line requires the user setting `http.proxyStrictSSL` to be set to false to pass.